### ISSUE TITLE
[QA] encryption type defaults to deprecated user key encryption

### ISSUE THREAD

**Issue by jnweiger** [owncloud/encryption#327]
Seen with encryption 1.5.1

1) admin goes to disabled apps and enables "Default encryption module"
2) admin goes to encryption and starts with the first item: Enable server side encryption.
![image](https://user-images.githubusercontent.com/1108546/155526884-4b39e188-968d-4ede-a8ab-6f8147c16842.png)
----
3) Several precautory warnings are given about the implictions of encryption, but masterkey vs. user key is not discussed:
![image](https://user-images.githubusercontent.com/1108546/155526934-30a694a8-c9c3-4fd4-bd82-21090546b390.png)
----
4) Encryption can be enabled, before choosing either master or user key:
![image](https://user-images.githubusercontent.com/1108546/155527192-0c11f5aa-24e7-4ec8-9672-69f37e80d25d.png)
----
5) When a user attempts to upload a file, he gets instruction to re-login.
![image](https://user-images.githubusercontent.com/1108546/155529594-618de97a-daf0-4574-80a1-ddc5883b9727.png)
----
6) After lougout, login, and a refresh, the admin learns, that actually user key encryption mode was chosen.
![image](https://user-images.githubusercontent.com/1108546/155527296-7d94e132-98ee-4315-9e0e-d0b178f5a1cb.png)
----

Expected behaviour:
 * default encryption mode must be master key.
 * (or the UI should insist that an explicit choice is made.)
 * UX: A user should never run into a bad situation, when working through a UI from top to bottom.

Workaround: 
* after the damage is done: none. it cannot be switched back to master key, afaik.
* Do not work from top to bottom in the UI - start with the second item and choose a mode.

**Comment by jnweiger** [comment]
Same behaviour, when encryption is enabled via occ:

```
$ occ encryption:enable
Encryption enabled

No encryption module is loaded
$ occ app:enable encryption
encryption enabled
```

Now the next user logout login selects user key mode.

**Comment by phil-davis** [comment]
I agree - we should make it really easy to get master-key encryption by default when enabling encryption, and make it much more difficult to select user-key encryption. New installations should never use user-key encryption. The commands for enabling it should only be there for some edge-cases where some installation has "accidentally" switched it off, and is trying to switch it back on and recover people's user-key-encrypted files (if there is even an edge case where that is useful/possible to get someone out of a problem)

**Comment by jnweiger** [comment]
Btw. This response from `occ encryption:enable` is self contradicting:
```
Encryption enabled

No encryption module is loaded
```

Suggest to say the opposite in this case: `Encrpytion **not** enabled.`

**Comment by phil-davis** [comment]
Also see issue https://github.com/owncloud/enterprise/issues/5038 and https://github.com/owncloud/core/issues/27181

**Comment by jnweiger** [comment]
Fixed in encryption 1.6.0
